change/@fluentui-react-tabster-b2d43df9-1387-475e-b4e3-a4204207aba2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26584&commitId=6306ae51549a9199b1cee4dfe4066664f6da7613&env=prod)

✅ There was 1 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Persona with children.default.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>